CONTRIBUTING.md

```diff
@@ -136,6 +136,23 @@ npm run test:coverage
 
 See the [Create-React-App docs](https://github.com/facebook/create-react-app) for further information on JS testing.
 
+#### Testing package imports
+
+You can simulate how the published package will behave when imported into another JavaScript application by running
+
+```bash
+npm run lib-test
+```
+
+This script uses `npm pack` to package Kedro-Viz as a tarball, then copies it to a boilerplate React app in `/tools/test-lib/react-app` and installs it. Next, the script runs a simple server at `http://localhost:1337` so you can check that everything is working as it should.
+
+You can also run automated tests on this demo app by navigating to this directory and running Jest:
```

Comment:
Is this something you need to install separately or is it already installed?

Maybe link to it https://www.npmjs.com/package/jest ? I had to check that Jest wasn't a typo for test (but that's because I no nothing of this subject, so probably wouldn't be a problem for most)

Reply:
It's installed automatically when you run the `npm run lib-test`, no extra setup required.

It's actually already mentioned and linked to above on line 119, so I don't think it's necessary to link it again? And Jest is a well-known and popular JS test-runner, so it's probably unnecessary to clarify.

Reply:
Yeah, it's fine! This is not documentation meant for me, so if it's obvious for the audience, all is good.